aws/table_aws_efs_access_point.go

```diff
@@ -0,0 +1,191 @@
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEfsAccessPoint(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_efs_access_point",
+		Description: "AWS EFS Access Point",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.SingleColumn("access_point_id"),
+			ShouldIgnoreError: isNotFoundError([]string{"AccessPointNotFound"}),
+			Hydrate:           getEfsAccessPoint,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listEfsAccessPoints,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the access point. This is the value of the Name tag.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "access_point_id",
+				Description: "The ID of the access point, assigned by Amazon EFS.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "access_point_arn",
+				Description: "The unique Amazon Resource Name (ARN) associated with the access point.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "life_cycle_state",
+				Description: "Identifies the lifecycle phase of the access point.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "file_system_id",
+				Description: "The ID of the EFS file system that the access point applies to.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "client_token",
+				Description: "The opaque string specified in the request to ensure idempotent creation.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "owner_id",
+				Description: "Identified the AWS account that owns the access point resource.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "posix_user",
+				Description: "The full POSIX identity, including the user ID, group ID, and secondary group IDs on the access point that is used for all file operations by NFS clients using the access point.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "root_directory",
+				Description: "The directory on the Amazon EFS file system that the access point exposes as the root directory to NFS clients using the access point.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "tags_src",
+				Description: "The tags associated with the access point, presented as an array of Tag objects.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns for all tables
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(getEfsAccessPointTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
```

Comment:
Is `Name` is a required field, or it can be null also?
If so, use `access_point_id` as title (if name is null)